### PR TITLE
alert when the testgrid config upload job fails

### PR DESCRIPTION
**Pull request by BenTheElder**

### CODE REVIEW

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -4137,6 +4138,8 @@ dashboards:
     test_group_name: maintenance-ci-testgrid-config-upload
     code_search_url_template:
       url: https://github.com/kubernetes/test-infra/compare/<start-custom-0>...<end-custom-0>
+    alert_options:
+      alert_mail_to_addresses: 'gke-kubernetes-engprod+alerts@google.com'
```
Comment (BenTheElder) [inline testgrid/config/config.yaml]: not sure which address we want to use, but *somebody* should get a ping when the config updater fails

Comment (krzyzacy) [reply testgrid/config/config.yaml]: feel free to add my personal email here :-)

**Pull request merged**